tests/test_ophys/test_tools_roiextractors.py

```diff
@@ -1427,6 +1469,25 @@ def test_add_one_photon_series_roundtrip(self):
             expected_one_photon_series_shape = (self.num_frames, self.num_columns, self.num_rows)
             assert one_photon_series.shape == expected_one_photon_series_shape
 
+    def test_add_multiple_one_photon_series_with_same_imaging_plane(self):
+        """Test adding two OnePhotonSeries that use the same ImagingPlane."""
+        add_photon_series(
```

Comment:
What do you think of making this explicit by explictly setting the:

```python
first_photon_series_metadata = deepcopy(self.one_photon_series_metadata)
second_photon_series_metadata = deepcopy(self.one_photon_series_metadata)
first_photon_series_metadata["Ophys"]["OnePhotonSeries"][0]["imaging_plane"] = "same_imaging_plane_for_two_series"
second_photon_series_metadata["Ophys"]["OnePhotonSeries"][0]["imaging_plane"] = "same_imaging_plane_for_two_series"
```
Or we could use a single metadata dictionary for the two of them as well...

Reply:
explicit is always better, but I'm not sure how you imagine the single metadata dictionary, something like this:
https://github.com/catalystneuro/neuroconv/blob/3c17f248da202be4320473d8ef71b4b4d021385c/tests/test_ophys/test_tools_roiextractors.py#L1474-L1480
https://github.com/catalystneuro/neuroconv/pull/594/commits/3c17f248da202be4320473d8ef71b4b4d021385c

Reply:
So,
`metadata["Ophys"]["OnePhotonSeries"]` will be a list with two elements (one per plane).

But if we are moving towards https://github.com/catalystneuro/neuroconv/issues/269 then maybe is not worth to do it like this.

Reply:
No, not all, I am saying that if you don't want to have a single metadata dictionary that is totally fine. Because, when we eventually do #269 then we will need to change the list structure anyway.

So up to you.